src/Symfony/Component/Process/Process.php

```diff
@@ -471,6 +478,9 @@ public function wait(?callable $callback = null): int
      * from the output in real-time while writing the standard input to the process.
      * It allows to have feedback from the independent process during execution.
      *
+     * @param (callable('out'|'err', string):void)|null $callback A PHP callback to run whenever there is some
+     *                                                            output available on STDOUT or STDERR
```

Comment:
@nicolas-grekas This one should return bool
```suggestion
     * @param (callable('out'|'err', string):bool)|null $callback A PHP callback to run whenever there is some
     *                                                            output available on STDOUT or STDERR
```

Reply:
indeed thanks!